tests/example.spec.ts

```diff
@@ -1,43 +1,52 @@
 import { test, expect } from '@playwright/test';
+import { faker } from '@faker-js/faker';
+
+const URL = process.env.APP_URL
 
 test.beforeEach(async ({ page }) => {
-  await page.goto("https://fe-delivery.tallinn-learning.ee/signin");
+  await page.goto(URL);
+
+
+  test('Check for incorrect credentials message and close pop up message', async ({page}) => {
+    const randomUsername = faker.internet.username()
+    const randomPassword = faker.internet.password();
+
+    const usernameField = page.getByTestId("username-input");
+    const passwordField = page.getByTestId("password-input");
+    const signInButton = page.getByTestId("signIn-button");
+    const errorPopUpMessage = page.getByTestId("authorizationError-popup");
+    const closeButton = page.getByTestId("authorizationError-popup-close-button");
+
+    await usernameField.fill(randomUsername);
+    console.log(randomUsername)
+    await passwordField.fill(randomPassword);
+    console.log(randomPassword)
+    await signInButton.click();
+    await expect(errorPopUpMessage).toBeVisible();
+
+    await closeButton.click();
+    await expect(signInButton).toBeEnabled();
+  });
+
+  test('Check for error messages for login input', async ({page}) => {
+    const usernameField = page.getByTestId("username-input");
+    const passwordField = page.getByTestId("password-input");
+    const emptyErrorMessageForUserName = page.getByTestId('username-input-error').nth(0);
+    const emptyErrorMessageForPassword = page.getByTestId('username-input-error').nth(1);
+
+    const emptyErrorMessageForShortUserName = page.getByText('The field must contain at least of characters: 2');
+    const emptyErrorMessageForShortPassword = page.getByText('The field must contain at least of characters: 8');
+
+    await usernameField.fill("t");
```

Comment:
Faker could also be used to generate a random too short username and password by specifying min and max number of characters.

Reply:
CI is not passing